jetbrains-rider/it/software/aws/toolkits/jetbrains/services/lambda/dotnet/DotnetLocalLambdaRunConfigurationIntegrationTest.kt

```diff
@@ -112,6 +114,26 @@ abstract class DotnetLocalLambdaRunConfigurationIntegrationTestBase(private val
             .containsEntry("AWS_SESSION_TOKEN", "")
     }
 
+    @Test
+    fun samIsExecutedWithFileInput() {
+        val input = Files.createTempFile(tempTestDirectory.toPath(), "tmp", null).also {
```

Comment:
Why is this file created differently?

Reply:
these tests use rider's test base which don't have a fixture

Reply:
Ah, fair